.gitlab-ci.yml

```diff
@@ -60,6 +60,12 @@ deploy-gcp-staging:
     #- docker login -u "gitlab-ci-token" -p "$CI_BUILD_TOKEN" $CI_REGISTRY
     - export HELM_REPLICA_COUNT=$HELM_REPLICA_COUNT_STAGING
     - export SECRET_KEY_BASE=$SECRET_KEY_BASE_STAGING
+    - export DATABASE_HOST=$GCP_DATABASE_HOST_STAGING
```

Comment:
we're using this convention for app env settings (numero-member app, for example), so let's stick to this current convention:

export <APP_ENV_KEY>=$<APP_ENV_KEY>_<ENV>

for example:

- export SECRET_KEY_BASE=$SECRET_KEY_BASE_STAGING_GCP
- export DATABASE_HOST=$DATABASE_HOST_STAGING_GCP

- export DATABASE_URI=$DATABASE_URI_PROD_GCP

Please apply this convention for all.